desktop/app/app-state.js.flow

```diff
@@ -0,0 +1,35 @@
+/* @flow */
+
+export type State = {
+  x: ?number,
+  y: ?number,
+  width: number,
+  height: number,
+  windowHidden: boolean,
+  isMaximized: ?boolean,
+  isFullScreen: ?boolean,
+  displayBounds: ?any,
+  tab: ?string,
+}
+
+export type Options = {
+  path: string,
+  eventHandlingDelay: number,
```

Comment:
oh right... k I split into Config and Options are whats passed into constructor is Options and and local config is in Config.